src/main/java/net/dv8tion/jda/api/interactions/commands/build/CommandData.java

```diff
@@ -51,37 +49,41 @@ public interface CommandData extends SerializableData
     CommandData setName(@Nonnull String name);
 
     /**
-     * Whether this command is available to everyone by default.
-     * <br>If this is disabled, you need to explicitly whitelist users and roles per guild.
+     * Sets the default {@link DefaultMemberPermissions} for this command.
+     * <br>By default, everyone can use this command ({@link DefaultMemberPermissions#ENABLED}). This can be configured or overwritten by moderators in each guild.
      *
-     * <p>You can use {@link CommandPrivilege} to enable or disable this command per guild for roles and members of the guild.
-     * See {@link Command#updatePrivileges(Guild, CommandPrivilege...)} and {@link Guild#updateCommandPrivileges(Map)}.
+     * <p>Passing {@link DefaultMemberPermissions#DISABLED} will only show this command to admins.
      *
-     * @param  enabled
-     *         True, if this command is enabled by default for everyone. (Default: true)
+     * @param  permission
+     *         {@link DefaultMemberPermissions} representing the default permissions of this command.
      *
      * @return The builder instance, for chaining
+     *
+     * @see DefaultMemberPermissions#ENABLED
+     * @see DefaultMemberPermissions#DISABLED
      */
     @Nonnull
-    CommandData setDefaultEnabled(boolean enabled);
+    CommandData setDefaultPermissions(@Nonnull DefaultMemberPermissions permission);
```

Comment:
```
Sets the {@link Permissions permissions} that a user/role must have in a specific channel to be able to use this command.
<br>By default, everyone can use this command ({@link DefaultMemberPermissions#ENABLED}). Additionally, a command can be disabled for everyone but admins via {@link DefaultMemberPermissions#DISABLED}.
<p>These configurations can be overwritten by moderators in each guild. See <whatever#getPrivledges> to get moderator defined overrides.
```

Here is some suggested rewording. This isn't necessarily perfect and needs to be tweaked, but i think this provides better context. Tweak as needed.

Reply:
```
Sets the {@link net.dv8tion.jda.api.Permission Permissions} that a user must have in a specific channel to be able to use this command.
<br>By default, everyone can use this command ({@link DefaultMemberPermissions#ENABLED}). Additionally, a command can be disabled for everyone but admins via {@link DefaultMemberPermissions#DISABLED}.
<p>These configurations can be overwritten by moderators in each guild. See {@link Command#retrievePrivileges(Guild)} to get moderator defined overrides.
```

That's what i came up with

Reply:
looks good to me